hr-panel/package.json

```diff
@@ -14,8 +15,9 @@
     "@testing-library/user-event": "^13.5.0",
     "@types/jest": "^27.5.2",
     "@types/node": "^16.18.80",
-    "@types/react": "^18.2.55",
+    "@types/react": "^18.2.56",
     "@types/react-dom": "^18.2.19",
```

Comment:
Move these packages from `dependencies` to `devDependencies` as these packages should not be included in deployment builds.